src/utils/showcase.ts

```diff
@@ -0,0 +1,77 @@
+import { VoteResult } from '../typings/interfaces'
+import Discord, { TextChannel } from 'discord.js'
+import safeSendMessage from '../utils/safeSendMessage'
+import createProjectEmbed from '../utils/createProjectEmbed'
+
+export interface ShowcaseInput {
+  result: VoteResult
+  isUpvote: boolean
+  isDownvote: boolean
+  isPause: boolean
+  guild: Discord.Guild
+  channel: Discord.TextChannel | Discord.DMChannel | Discord.NewsChannel
+  user: Discord.User
+  reaction: Discord.MessageReaction
+}
+
+export default async function (input: ShowcaseInput): Promise<Discord.Message | undefined> {
+  const { result, guild, channel, user, reaction, isUpvote, isDownvote, isPause } = input
+  const { success, reason, project } = result
+
+  // Alert of errors during the vote process
+  if (!success || !project) {
+    log.error(`Could not register ${user.id}'s vote for project ${project?.name} (ID ${project?.id}): ${reason}`)
+    return await safeSendMessage(channel, '⚠️ Your vote was not possible to register. (Internal error)')
+  }
+
+  // Only one of these can be true per operation depending on what reaction was used
+  const wasApproved = result?.wasApproved === true
+  const wasRejected = result?.wasRejected === true
+  const wasPaused = result?.wasPaused === true
+
+  if (isPause && !isUpvote && !isDownvote) {
+    log.info(`Project ${project.name} (ID ${project.id}) was ${wasPaused ? 'paused' : 'resumed'} by **${user.tag}** (${user.id})`)
```

Comment:
There appears to have been a bit of a mix-up here, considering the log appears to be trying to use Discord Markdown syntax despite obviously being sent into stdout. It could also use some wording improvements.
```suggestion
    log.info(`Voting on project ${project.name} (ID ${project.id}) was ${wasPaused ? 'suspended' : 'unsuspended'} by user ${user.id} (${user.tag})`)
```